<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newDashboard</class>
 <widget class="QMainWindow" name="newDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="frMain">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">/*-------------------------------------------------- Frames --------------------------------------------------*/
#frCabecalho {
	background-color: #3F4E8C;
}

#frBottom {
	background-color: #3A405A;
}

#frAvatar {
	background-color: #3F64FF;
	border-radius: 20px;
}

/*-------------------------------------------------- Labels --------------------------------------------------*/
QLabel {
	color: white;
}

#lbNomeEscritorio {
	font: 24pt &quot;Bebas Neue&quot;;
}

#lbNomeAdvogado {
	font: 10pt &quot;Avenir LT Std&quot;;
}

#lbOAB {
	font: 8pt &quot;Avenir LT Std&quot;;
	font-weight: 750;
}

/*-------------------------------------------------- Push Buttom --------------------------------------------------*/
#pbMenuPrincipal {
	background-color: #292D40;
	border: 0px solid transparent;
	border-radius: 21px;
	
	background-image: url(:/menuPrincipal/hamburger.png);
	background-repeat: no-repeat;
	background-position: center;
}

#pbAvisos {
	background-color: transparent;

	background-image: url(:/menuPrincipal/avisos.png);
	background-repeat: no-repeat;
	background-position: center;
}

/*-------------------------------------------------- Stacked Eidget --------------------------------------------------*/
#stkPrincipal {
	background-color: white;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QFrame" name="frCenter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frCabecalho">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QFrame" name="frInfoAdv">
            <property name="maximumSize">
             <size>
              <width>230</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0" rowspan="2" alignment="Qt::AlignLeft|Qt::AlignVCenter">
              <widget class="QFrame" name="frAvatar">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>40</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>IA</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1" alignment="Qt::AlignLeft|Qt::AlignBottom">
              <widget class="QLabel" name="lbNomeAdvogado">
               <property name="text">
                <string>Nome do Advogado</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignTop">
              <widget class="QLabel" name="lbOAB">
               <property name="text">
                <string>OAB: 99999/SP</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QLabel" name="lbNomeEscritorio">
            <property name="text">
             <string>Domin√°ria e associados</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbAvisos">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbMenuPrincipal">
            <property name="minimumSize">
             <size>
              <width>42</width>
              <height>42</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>42</width>
              <height>42</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frConteudo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QStackedWidget" name="stkPrincipal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frBottom">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Resources/newDashboard.qrc"/>
 </resources>
 <connections/>
</ui>
